terraform/modules/prom-ec2/alerts-config/govuk-accounts-alerts.yml

```diff
@@ -0,0 +1,21 @@
+groups:
+- name: GovukAccounts
+  rules:
+  - alert: GovukAccounts_AppRequestsExcess5xx
+    expr: sum without(exported_instance, status_range) (rate(requests{org="govuk-accounts", space="production", status_range="5xx"}[5m])) / sum without(exported_instance, status_range) (rate(requests{org="govuk-accounts", space="production"}[5m])) >= 0.25
+    for: 120s
+    labels:
+        product: "govuk-accounts"
+    annotations:
+        summary: "App {{ $labels.app }} has too many 5xx errors"
```

Comment:
Should this (and elsewhere) be `labels.product`?

Reply:
`labels.app` will tell us whether this is the account manager or attribute service, whereas `labels.product` will just say `govuk-accounts` ([here's an example in Prometheus dashboard](https://prom-2.monitoring.gds-reliability.engineering/graph?g0.range_input=1h&g0.expr=sum%20without(exported_instance%2C%20status_range)%20(rate(requests%7Borg%3D%22govuk-accounts%22%2C%20space%3D%22production%22%2C%20status_range%3D%225xx%22%7D%5B5m%5D))%20%2F%20sum%20without(exported_instance%2C%20status_range)%20(rate(requests%7Borg%3D%22govuk-accounts%22%2C%20space%3D%22production%22%7D%5B5m%5D))&g0.tab=1)).